doc/developer-guides/hld/hv-memmgt.rst

```diff
@@ -52,16 +52,18 @@ inside the hypervisor and from a VM:
 
 -  How ACRN hypervisor manages host memory (HPA/HVA)
 -  How ACRN hypervisor manages the Service VM guest memory (HPA/GPA)
--  How ACRN hypervisor and the Service VM DM manage the User MV guest memory (HPA/GPA)
+-  How ACRN hypervisor and the Service VM Device Model (DM) manage the User VM
+   guest memory (HPA/GPA)
 
 Hypervisor Physical Memory Management
 *************************************
 
 In ACRN, the HV initializes MMU page tables to manage all physical
 memory and then switches to the new MMU page tables. After MMU page
 tables are initialized at the platform initialization stage, no updates
-are made for MMU page tables except when hv_access_memory_region_update is called.
-However, the memory region updated by hv_access_memory_region_update
+are made for MMU page tables except when ``hv_access_memory_region_update`` is
+called.
+However, the memory region updated by ``hv_access_memory_region_update``
```

Comment:
Where is "hv_access_memory_region_update" in the code?

Reply:
That function existed in mmu.h in v2.2 but not in v2.3, and it looks like it was renamed to ``ppt_clear_user_bit`` as indicated in this commit title:
```
f3067f538 hv: mmu: rename hv_access_memory_region_update to ppt_clear_user_bit
```
But now that function doesn't seem to exist any more either; it disappeared with v2.5.  @NanlinXie can you help figure out how to update this documentation that mentions ``hv_access_memory_region_update``?

Reply:
@lifeix , please help to update HLD.

Reply:
@amyreye  Would you please help to the below update:
-are made for MMU page tables except when hv_access_memory_region_update is called.
-However, the memory region updated by hv_access_memory_region_update
+are made for MMU page tables except when set_paging_supervisor/nx/x is called.
+However, the memory region updated by set_paging_supervisor/nx/x